src/libraries/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -51,7 +51,10 @@ private static void ConcurrentInitializeTest()
                     {
                         countdownEvent.Signal();
                         countdownEvent.Wait(ThreadTestHelpers.UnexpectedTimeoutMilliseconds);
-                        Assert.True(ThreadPool.SetMinThreads(processorCount, processorCount));
+                        if (Boolean.Parse(UsePortableThreadPool))
+                        {
+                            Assert.True(ThreadPool.SetMinThreads(processorCount, processorCount));
+                        }
```

Comment:
Is it necessary to pass in the UsePortableThreadPool value to the remote process? I was expecting that the remote process would also pick up the runtime config option set in the project, and that we should just be able to use the static UsePortableThreadPool property from the delegate.

Reply:
Yea I was expecting the same, not sure why but it wasn't picking up the config option, it was just evaluating to false in any case so I had to pass it as an argument